readthedocs/projects/migrations/0020_add-api-project-proxy.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-10-27 12:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0019_add-features'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='APIProject',
+            fields=[
+            ],
+            options={
+                'proxy': True,
+            },
+            bases=('projects.project',),
+        ),
```

Comment:
Curious that Django makes a migration for a proxy model. I assume it doesn't do anything to the DB?

Reply:
https://stackoverflow.com/questions/37988914/why-does-django-create-migration-files-for-proxy-models seems to say it's so later migrations can reference it.

Reply:
Yeah, i was surprised by this too at first, i didn't realize this when doing the integrations proxy models.